### DISCUSSION TITLE
Password not encrypted from client

### DISCUSSION THREAD

**Discussion by arilanto** [strapi/strapi#7170] [Legacy Topics]
Hello,

I'm wondering if it is normal but when I use auth/local to login, I can see my password 'in clear' in my browser's request header, so it means that anybody can intercept my passwork through the internet ?
User's password is encrypted by strapi in the database after creating user, so the same encryption should be done on the client side before calling auth/local ? but If I do so, strapi will not recognise my password as it expects decrypted password from client.

Thank u,

**Comment by lauriejim** [comment]
Hello! That's completely normal.
You will see it's the same with every other softwares / services you are using.
To secure the requisition, you will have to use an SSL certificate to have a HTTPS request.
This will secure the thing you are talking about.

**Reply by arilanto** [reply]
Thank u @lauriejim 
but I still think that encryption should be done on client side even https is up
